src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/MultiBindingCondition.shared.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Xamarin.CommunityToolkit.Helpers
+{
+	public enum MultiBindingCondition
```

Comment:
```[Flags]``` attribute is missed if we want to support multiple options)

Reply:
But why would you? They all kind of mutually exclude one another right?

Reply:
Perhaps it's useless, but I don't think they all exclude others.

4 converters and I want to say that **valid** ALL or None.

But maybe let's skip it for now